modules/react-components/src/components/page-header/page-header.tsx

```diff
@@ -131,6 +140,67 @@ export const PageHeader: React.FunctionComponent<PageHeaderPropsInterface> = (
             "fluid": isLoading
         }
     );
+    
+    const headingContent = (
+        <div className={ innerClasses }>
+            { image && (
+                <GenericIcon
+                    icon={
+                        isLoading ?
+                            (
+                                <div className="fluid">
+                                    <Placeholder style={ { height: 100, width: 100 } }>
+                                        <Placeholder.Image square/>
+                                    </Placeholder>
+                                </div>
+                            )
+                            : image
+                    }
+                    size="tiny"
+                    transparent
+                    spaced="right"
+                    data-testid={ `${ testId }-image` }
+                />
+            ) }
+
+            {
+                isLoading
+                    ? (
+                        <Header className="page-header ellipsis"
+                                as={ titleAs }
+                                textAlign={ titleTextAlign }
+                        >
+                            <div style={ { width: "250px" } }>
```

Comment:
Should we remove this inline style?

Reply:
This is used for the loading placeholders. I don't see a advantage in writing a CSS class for this. WDYT?

Reply:
If we aren't using it anywhere else that's fine I guess.

Reply:
This placeholder will only be used for this component. (y)